components/zcash_protocol/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+## [0.4.1] - 2024-11-13
```

Comment:
Hmm, we bump MSRV in this release, which I usually try to avoid doing in a point release (because there's no way to subdivide point releases, meaning if we ever needed to backport a bugfix we wouldn't be able to do so to pre-MSRV-bump downstreams).